### ISSUE TITLE
how to set the parameters for training

### ISSUE THREAD

**Issue by BoXiao123** [marvis/pytorch-mobilenet#10]
Hi @marvis thanks for your job
I want to train the mobilenet on Imagenet by myself. Could you please indicate how to set the parameters (like  batch_size,lr and so on) in order to achieve the similar accuracy.Thanks

**Comment by Coderx7** [comment]
@BoXiao123 : I guess you know this by now that everything is provided in the `main.py file` . using that you can train this on imageNet.
by the way, I dont know if you have already trained this or not, but if you have, i'd appreciate any feedback.
I have been using the same config as given in the main.py but with a few changes, instead of batch of 256, I used batch of 128. and I also set` nesterov=True`  like this : 
```
 optimizer = torch.optim.SGD(model.parameters(), args.lr,
               momentum=args.momentum,
               weight_decay=args.weight_decay,
               nesterov=True)
```
now the problem is at 123 epoch my accuracy is `64.48/85.95` which is very bad!! compared to 69% acc.
I asked @marvis but he has not yet answered, so if you trained it yourself, would you report back your results as well?

**Comment by BoXiao123** [comment]
Hi @Coderx7 ,
Actually I didn't trained on imageNet. I tried to train mobilenet via different platforms. Only tensorflow-mobilenet can reproduce the acc.

**Comment by Coderx7** [comment]
@BoXiao123 Hi and Thank you very much, your feedback is really appreciated my friend.

**Comment by tzc1994** [comment]
Hi @BoXiao123 , Do you slove this problem? What hyperparameteres are? I only achieve top1 at 43% on ImageNet12. Thanks!

**Comment by notabigfish** [comment]
@Coderx7 Hi, I've trained the model on ImageNet12 with 256 batchsize and set nesterov=False, and got 68.454% top1 acc. However, it still did not reach the best acc. Do you know how the rmsprop init from sgd works? Maybe this is the trick for higher accuracy.

**Comment by Coderx7** [comment]
if you train it for like 120 epochs, you should be able to achieve 70% top1 or around that. (using simple sgd with steplr scheduler that decreases learning rate every 30 epochs.
